lms/djangoapps/course_home_api/outline/v1/tests/test_views.py

```diff
@@ -366,3 +369,39 @@ def test_verified_mode(self):
 
         response = self.client.get(self.url)
         assert response.data['verified_mode'] == {'access_expiration_date': (enrollment.created + MIN_DURATION), 'currency': 'USD', 'currency_symbol': '$', 'price': 149, 'sku': 'ABCD1234', 'upgrade_url': '/dashboard'}
+
+    @override_experiment_waffle_flag(COURSE_HOME_MICROFRONTEND, active=True)
+    def test_hide_learning_sequences(self):
+        """
+        Check that Learning Sequences filters out sequences.
+        """
+        CourseEnrollment.enroll(self.user, self.course.id, CourseMode.VERIFIED)
+
+        # Normal behavior: the sequence exists
+        response = self.client.get(self.url)
+        assert response.status_code == 200
+        blocks = response.data['course_blocks']['blocks']
+        seq_block_id = next(
+            block_id
+            for block_id, block in blocks.items()
+            if block['type'] == 'sequential'
+        )
+
+        # With Learning Sequences active and a course outline loaded, the same
+        # sequence is removed.
+        with override_waffle_flag(USE_FOR_OUTLINES, active=True):
+            new_learning_seq_outline = CourseOutlineData(
+                course_key=self.course.id,
+                title="Test Course Outline!",
+                published_at=datetime(2021, 6, 14, tzinfo=timezone.utc),
+                published_version="5ebece4b69dd593d82fe2022",
+                entrance_exam_id=None,
+                days_early_for_beta=None,
+                sections=[],
+                self_paced=False,
+                course_visibility=CourseVisibility.PRIVATE
+            )
+            replace_course_outline(new_learning_seq_outline)
+            response = self.client.get(self.url)
+            blocks = response.data['course_blocks']['blocks']
+            assert seq_block_id not in blocks
```

Comment:
Question: And it's removed because the course outline is empty?

Reply:
Yes.